src/test/java/org/htmlunit/html/HtmlImageTest.java

```diff
@@ -318,4 +318,44 @@ public void clickWithCoordinates() throws Exception {
         img.click(2, 7);
         assertEquals(getExpectedAlerts()[0] + getExpectedAlerts()[1], page.getTitleText());
     }
+
+    /**
+     * @throws Exception on test failure
+     */
+    @Test
+    @Alerts({"begin-end-onload-", "onclick-begin-end-onload-", "onmousedown-onmouseup-begin-end-onclick-onload-"})
+    public void onload() throws Exception {
+        final String html = ""
+                + "<html>\n"
+                + "<head>\n"
+                + "<script>\n"
+                + "  function log(msg) { window.document.title += msg + '-';}\n"
+                + "  function loadImage() {\n"
+                + "    log('begin');\n"
+                + "    var image = new Image(4, 7);\n"
+                + "    image.onload = function () { log(\"onload\") };\n"
+                + "    image.src = '4x7.jpg';\n"
+                + "    log('end');\n"
+                + "  }\n"
+                + "</script>\n"
+                + "</head>\n"
+                + "<body onload=\"loadImage();\">\n"
```

Comment:
don't really need these verbose characters
```suggestion
                + "<body onload=\"loadImage()\">\n"
```